pkg/applicationserver/config.go

```diff
@@ -309,3 +310,20 @@ func (c EndDeviceLocationStorageConfig) NewRegistry(ctx context.Context, comp *c
 	}
 	return registry, nil
 }
+
+// LastSeenConfig defines configuration for the device last seen map which stores timestamps for batch updates.
+type LastSeenConfig struct {
+	MapCapacity int           `name:"map-capacity" description:"Capacity of last seen map"`
+	Interval    time.Duration `name:"flush-interval" description:"Interval at which lat seen timestamps are sent to Identity Server"`
+}
+
+// NewLastSeen defines a new batch update map.
+func (c LastSeenConfig) NewLastSeen(ctx context.Context, comp *component.Component) (*LastSeenProvider, error) {
+	if c.MapCapacity == 0 {
```

Comment:
I would prefer this to be explicit default configuration (since it is visible when you use `--help` or `config`).

Reply:
Or batching should be disabled on `0`.